project_testing/housing_characteristics/Census Region.tsv

```diff
@@ -0,0 +1,7 @@
+Dependency=County	Option=Midwest	Option=Northeast	Option=South	Option=West	sampling_probability
+AZ, Maricopa County	0.000000	0.000000	0.000000	1.000000	0.166667
+CA, Los Angeles County	0.000000	0.000000	0.000000	1.000000	0.166667
+GA, Fulton County	0.000000	0.000000	1.000000	0.000000	0.166667
+IL, Cook County	1.000000	0.000000	0.000000	0.000000	0.166667
+TX, Harris County	0.000000	0.000000	1.000000	0.000000	0.166667
+WA, King County	0.000000	0.000000	0.000000	1.000000	0.166667
```

Comment:
I don't remember how we ended up with these 6 locations but can we revise them? Right now we have 1 midwest, 0 northeast, 2 south, and 3 west. What if we switch to 4 (or a multiple of 4) locations, such that these regions are equally represented?

Reply:
@shorowit How about:
County | Census Region
-- | --
AZ, Maricopa County | West
IL, Cook County | Midwest
TX, Harris County | South
NY, New York County | Northeast

Reply:
Can you post the IECC climate zones for them? I'm guessing that NY & Cook County are similar climate zones, and same for Maricopa and Harris. It'd be nice to get more diversity of climate zones.

Reply:
I updated the comment above. Perhaps we'd want one county per IECC climate zone (1A - 7B; 15 locations)?

Reply:
I don't have a strong opinion on this, but I think our requirements are that we have a diversity of climate zones and an equal number of locations in each Census Region. If 4 locations, that would be Harris County, LA, NY, and Cook County. But if you wanted to add more, I wouldn't object.